packages/ui/src/layout/wui-flex/styles.ts

```diff
@@ -0,0 +1,7 @@
+import { css } from 'lit'
+
+export default css`
+  div {
+    display: flex;
```

Comment:
As we are keeping div in, I think we need to add `width: inherit` and `height: inherit` here, so that we can add styles on `wui-flex` from other places.